Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3.11.4
+
+WORKDIR /app
+
+RUN pip install flask
```

Comment:
Please move the libraries to the separate requirements.txt file and add 
RUN python -m pip install -r /requirements-dev.txt in the Dockerfile